lib/puppetserver/ca/utils/http_client.rb

```diff
@@ -196,7 +196,12 @@ def self.check_server_online(settings, logger)
             if e.wrapped.is_a? Errno::ECONNREFUSED
               return false
             else
-              raise e
+              if suppress_error
+                logger.inform("Could not determine that puppetserver is offline, " \
+                  "connection check failed with #{e.wrapped}. Proceeding with cert signing.")
```

Comment:
Since this check is used by various actions, I don't think this log message belongs here. It might make more sense for the caller to rescue this error if `--force` is used.

Reply:
I was actually wondering if we wanted the `force` flag to be usable with those other actions also, but assumed it was out of scope for now. Since currently that flag can only come from the `sign` command, it seemed okay to have the logging be specific to that.

Reply:
yeah idk it just doesn't seem great to me, conceptually. I would be okay with leaving _some_ logging here, but I think the "Proceeding with cert signing." bit doesn't belong here.

Reply:
also, I think this conditional needs to `return false`?